### PR TITLE
[ODS-4391] Convert EdFi.Common.UnitTests to .NET Core

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: Application/Ed-Fi-Ods.sln [hunk 1]
```diff
@@ -272,8 +274,8 @@ Global
 		{DFD89624-8424-4878-8DF4-3C7C449AF990}.Release|Any CPU.Build.0 = Release|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Npgsql|Any CPU
-		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.Build.0 = Npgsql|Any CPU
+		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.ActiveCfg = Release|Any CPU
+		{6371DB70-4AC7-48F3-AFE2-C87736FCEB9A}.Npgsql|Any CPU.Build.0 = Release|Any CPU
```
Comment (jamessmckay) [inline Application/Ed-Fi-Ods.sln:278]: what is driving this change?

File: Application/Ed-Fi-Ods.sln [hunk 2]
```diff
@@ -348,12 +350,12 @@ Global
...
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Npgsql|Any CPU.ActiveCfg = Debug|Any CPU
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Npgsql|Any CPU.Build.0 = Debug|Any CPU
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{C8DF24F4-05E0-4465-94AF-BFE9B8652E18}.Release|Any CPU.Build.0 = Release|Any CPU
+		{37CB77F7-1C51-480C-B536-CDBF5DE26E10}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{37CB77F7-1C51-480C-B536-CDBF5DE26E10}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{37CB77F7-1C51-480C-B536-CDBF5DE26E10}.Npgsql|Any CPU.ActiveCfg = Release|Any CPU
+		{37CB77F7-1C51-480C-B536-CDBF5DE26E10}.Npgsql|Any CPU.Build.0 = Release|Any CPU
```
Comment (jamessmckay) [inline Application/Ed-Fi-Ods.sln:356]: should this be debug?

Comment (raghuramakrishna) [reply Application/Ed-Fi-Ods.sln]: reverted

File: Application/Ed-Fi-Ods.sln (hunk 1 above)
Comment (raghuramakrishna) [reply Application/Ed-Fi-Ods.sln:278]: whenever we rebuild the solution, this is getting reset

**Pull request merged**